test/browser/features/HappyPath/BAVAccountDetailsFailOne.feature

```diff
@@ -0,0 +1,14 @@
+@success
+Feature: Provided Bank Details Success
```

Comment:
[nit] we call it bank details and account details, could we use consistent naming please just to avoid any confusion?